### DISCUSSION TITLE
enable painless regex

### DISCUSSION THREAD

**Discussion by rijafida** [Security-Onion-Solutions/securityonion#3797] [Unsupported Versions] [answered]
Hi,

I am a newb to security onion and learning on the go i want to add a scripted field in kibana that req regex based extraction.. i know i have to enable the following in elasticsearch.yml 

Script.painless.regex.enabled : true 
My elasticsearch.yml file exists in /root/SecurityOnion/salt/elasticsearch/files
Now i have edited it to include the above mentioned line but it still is not enabled i have checked thru GET statement i am very confused ... and quite stuck

**Comment by weslambert** [comment, marked as answer]
See: https://github.com/Security-Onion-Solutions/securityonion/discussions/3896#discussioncomment-639731
